packages/one-app-bundler/webpack/loaders/common.js

```diff
@@ -39,30 +39,72 @@ const cssLoader = ({ name = '', importLoaders = 2 } = {}) => ({
 
 /* eslint-disable inclusive-language/use-inclusive-words --
 config options for a third party library */
-const purgeCssLoader = () => {
-  const configOptions = getConfigOptions();
-  const whitelistPatterns = configOptions.purgecss.whitelistPatterns
-    ? configOptions.purgecss.whitelistPatterns.map((pattern) => new RegExp(pattern, 'i'))
+// transform strings deep and greedy parameters into regex
+const reconcileSafeList = (safelist) => {
+  if (!safelist || Array.isArray(safelist)) {
+    return safelist;
+  }
```

Comment:
I'd prefer this be broken out into 2 if statements. The intent is different, even though the result is the same.

Example
```
if(!safelist) return null;
```

Reply:
something to add here @Matthew-Mallimo, functions should not be called where the base condition is a validation/check and the function isn't recursive, instead, the check has to be performed before calling it, in this case, in line 63, `const safelist = Array.isArray(safelist) ? reconcileSafeList(purgecss.safelist) : WHATEVER VALUE WE NEED;`

Reply:
I addressed both concerns by wrapping the call to ```reconcileSafelist``` in that if statement inverted. I also decided to keep that if statement the same because i feel that now both conditions are of common intent to typecheck before calling the function.